qiskit/transpiler/preset_passmanagers/plugin.py

```diff
@@ -301,3 +302,36 @@ def list_stage_plugins(stage_name: str) -> List[str]:
         return plugin_mgr.scheduling_plugins.names()
     else:
         raise TranspilerError(f"Invalid stage name: {stage_name}")
+
+
+def passmanager_stage_plugins(stage: str) -> Dict[str, PassManagerStagePlugin]:
+    """Return a dict with, for each stage name, the class type of the plugin.
+
+    This function is useful for getting more information about a plugin:
+
+        from qiskit.transpiler.preset_passmanagers.plugin import passmanager_stage_plugins
+        routing_plugins = passmanager_stage_plugins('routing')
+        basic_plugin = routing_plugins['basic']
+        ?basic_plugin
```

Comment:
This `?` syntax is specific to IPython - you maybe want to use the standard Python built-in `help` instead.

Reply:
fixed in 6135d9f6c846dcc9f0671ef5a1a1971166975011